### PR TITLE
Extruder Calibration tweaks

### PR DESCRIPTION
**Pull request by michalprusa**

### CODE REVIEW

File: Firmware/langtool.pl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 use strict;
 use warnings;
 
-my @langs = ("en","cz","it","es","pl","de");
+my @langs = ("en","cz");
```
Comment (3d-gussner) [inline Firmware/langtool.pl:7]: Why disable other languages in MK2 branch?

**Review by 3d-gussner — COMMENTED** [review]
Disabled languages not needed in MK2 branch.
BTW. How is it going with the extra flash on the EINSY to offload languages and other stuff?

File: Firmware/variants/1_75mm_MK2-RAMBo13a-E3Dv6full.h [hunk 2]
```diff
@@ -13,7 +15,7 @@ GENERAL SETTINGS
 #define DEVELOPER
 
 // Printer name
-#define CUSTOM_MENDEL_NAME "Prusa i3 MK2"
+#define CUSTOM_MENDEL_NAME "Prusa ECal MK2"
```
Comment (zerg960) [inline Firmware/variants/1_75mm_MK2-RAMBo13a-E3Dv6full.h:18]: Why change the name?

**Comment by 3d-gussner** [comment]
@michalprusa I think this PR is outdated and can be closed.

**Pull request closed without merging**